test/rules/trailing-comma-interface/test.ts.lint

```diff
@@ -1,7 +1,7 @@
 export interface Foo {
   curly?: string[];
-  ~~~~~~~~~~~~~~~~~  [Trailing comma is required in interfaces properties.]
+                  ~ [0]
   larry: number;
-  ~~~~~~~~~~~~~~  [Trailing comma is required in interfaces properties.]
-  moe: number,
```

Comment:
Why is `moe` deleted?  I asked for the comma to be removed from the end of its line.  This will verify that omitted commas are added by the fixer.